.vib/ghost/goss/goss.yaml

```diff
@@ -0,0 +1,26 @@
+command:
+  which-ghost:
+    exec: which ghost
+    exit-status: 0
+    stdout:
+      - "/opt/bitnami/ghost/bin/ghost"
+    stderr: [ ]
+    timeout: 40000
+  ghost-version-ok:
+    exec: ghost version | grep --quiet "$APP_VERSION"
+    exit-status: 0
+    stdout: [ ]
```

Comment:
Why do we need the `stdout` here, if we are not verifying the output?

Reply:
I reckon this in fact validates that the output is empty. I mean, if there were an error and it was printed in `stdout`, this will fail as the output contains text.